external/js/cothority/src/byzcoin/proto/requests.ts

```diff
@@ -224,6 +224,55 @@ export class GetSignerCountersResponse extends Message<GetSignerCountersResponse
     }
 }
 
+/**
+ * Holds an instanceID and it's latest known version.
+ */
+export class IDVersion extends Message<IDVersion> {
+    static register() {
+        registerMessage("IDVersion", IDVersion);
+    }
+
+    readonly id: Buffer;
+    readonly version: Long;
```

Comment:
To be strict, every field set by `Message` can be `undefined` (that's the burden of ProtocolBuffer), so better type it correctly.

Reply:
Not sure - if the field is required, then protobuf should throw an error if the field is not there.